dll/cpl/sysdm/lang/bg-BG.rc

```diff
@@ -258,10 +258,11 @@ BEGIN
 END
 
 IDD_ENVIRONMENT_VARIABLES DIALOGEX 6, 18, 252, 245
-STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+STYLE DS_CENTER | DS_MODALFRAME | DS_SHELLFONT | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE | WS_THICKFRAME
```

Comment:
Please use the following ordering (same comment for the other translations):
```suggestion
STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CENTER | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE | WS_THICKFRAME
```
(also this will unify with the other environment-edit dialogs).